tasks/lint_rules2/src/eslint-rules.cjs

```diff
@@ -201,10 +198,50 @@ exports.loadPluginReactPerfRules = (linter) => {
 };
 
 /** @param {import("eslint").Linter} linter */
-exports.loadPluginNextRules = (linter) => {
+const loadPluginNextRules = (linter) => {
   for (const [name, rule] of Object.entries(pluginNextAllRules)) {
     const prefixedName = `nextjs/${name}`;
 
     linter.defineRule(prefixedName, rule);
   }
 };
+
+/**
+ * @typedef {{
+ *   npm: string;
+ *   issueNo: number;
+ * }} TargetPluginMeta
+ * @type {Map<string, TargetPluginMeta>}
+ */
+exports.ALL_TARGET_PLUGINS = new Map([
+  ["eslint", { npm: "eslint", issueNo: 479 }],
+  ["typescript", { npm: "@typescript-eslint/eslint-plugin", issueNo: 503 }],
+  ["n", { npm: "eslint-plugin-n", issueNo: 493 }],
+  ["unicorn", { npm: "eslint-plugin-unicorn", issueNo: 684 }],
+  ["jsdoc", { npm: "eslint-plugin-jsdoc", issueNo: 1170 }],
+  ["import", { npm: "eslint-plugin-import", issueNo: -1 }], // TODO: Fill issueNo
+  ["jsx-a11y", { npm: "eslint-plugin-jsx-a11y", issueNo: 1141 }],
+  ["jest", { npm: "eslint-plugin-jest", issueNo: 492 }],
+  ["react", { npm: "eslint-plugin-react", issueNo: 1022 }],
+  ["react-hooks", { npm: "eslint-plugin-react-hooks", issueNo: -1 }], // TODO: Fill issueNo
```

Comment:
@Boshen  Would you create new issues for them? Or should I? 👀

Reply:
https://github.com/oxc-project/oxc/issues/2174